karman-avionics/src/utils/CC2500_regdefs.h

```diff
@@ -1,90 +0,0 @@
-/* Address Config = No address check */
```

Comment:
These are marked as autogenerated files. Did you intentionally delete them? And if not will it fuck with stuff if we check it in?

Reply:
This file is unnecessary until we go back to targeting the CC2500 radio module. For now, it's not needed.